pleskdistup/main.py

```diff
@@ -356,13 +357,20 @@ def main():
         "-v", "--version", action="store_true",
         help="show the version of this utility."
     )
+    parser.add_argument("--locale", default="en_US.UTF-8", help="set the locale for the utility")
```

Comment:
We need to carry this option to resume stage, otherwise the utility can fail after reboot.